api/app/serializers/spree/v2/storefront/taxon_serializer.rb

```diff
@@ -0,0 +1,13 @@
+module Spree
+  module V2
+    module Storefront
+      class TaxonSerializer < BaseSerializer
+        set_type :taxon
+        attributes :id, :parent_id, :position, :name, :permalink, :taxonomy_id,
+                   :lft, :rgt, :description, :meta_title, :meta_description,
+                   :meta_keywords
+        belongs_to :taxonomy
+      end
```

Comment:
I think that it would be good to add belongs to parent and `has_many` for children. They're not explicitly specified in model but they come from `awesome_nested_set` so they exist in model